indico/web/client/js/react/components/index.js

```diff
@@ -56,7 +56,7 @@ export {default as WTFPrincipalListField} from './WTFPrincipalListField';
 export {default as WTFPrincipalField} from './WTFPrincipalField';
 export {default as WTFLinkListField} from './WTFLinkListField';
 export {default as WTFSUIColorPicker} from './WTFSUIColorPicker';
-export {default as WTFSearchDropdown} from './WTFSearchDropdown';
+export {default as WTFSearchDropdown, SearchDropdown} from './WTFSearchDropdown';
```

Comment:
I guess importing it from `indico/react/components/WTFSearchDropdown` does not work?

Reply:
We were getting a rather mixed behaviour importing this component, where we could import it like you said with no problems in one file but it gives an error in other places. I think it's related the bundles, for example the import works when used in a file injected in this bundle: `self.inject_bundle('index.js')` but it does not work when imported in a file for a specific WP e.g. `self.inject_bundle('registration-form.js', WPManageRegistration)`